tests/Aspire.Playground.Tests/ProjectSpecificTests.cs

```diff
@@ -76,7 +76,29 @@ await WaitForAllTextAsync(app,
             timeoutSecs: 160);
 
         // Assert that HTTP triggers work correctly
-        await AppHostTests.CreateHttpClientWithResilience(app, "funcapp").GetAsync("/api/injected-resources");
+        var response = await AppHostTests.CreateHttpClientWithResilience(app, "funcapp").GetAsync("/api/injected-resources");
+
+        var output = await response.Content.ReadAsStringAsync();
+        /*
+            The output is expected to look something like this:
```

Comment:
This comment is duplicated with the expectedStrings below. Is the comment necessary?

Reply:
I reworked it